src/types.ts

```diff
@@ -1,6 +1,19 @@
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export type Extra = Map<string, any>;
 
+export interface Vector3D {
+  x: number;
+  y: number;
+  z: number;
+}
+
+export interface Orientation {
+  ox: number;
+  oy: number;
+  oz: number;
+  theta: number;
+}
```

Comment:
defined outside of movementsensor client since these interfaces will be used by other component signatures soon